src/components/movie-list/movie-list.ts

```diff
@@ -17,21 +18,27 @@ export class MovieList {
       return;
     }
 
-    this.container!.innerHTML = `
-      <div class="info">
-        <img src=${data.Poster} class="poster">
+    Array.from(data as any).filter((el: any) => {
```

Comment:
I think we already have data as Array and we don't need to wrap with this method here. Besides it would be great to get rid of any here as well. You can create small response model